src/Tizen.Multimedia.MediaPlayer/Player/AudioOffload.cs

```diff
@@ -47,7 +47,7 @@ internal AudioOffload(Player player)
 
         internal void CheckDisabled()
         {
-            if (Player.AudioOffload.IsEnabled)
+            if (Features.IsSupported(PlayerFeatures.AudioOffload) && Player.AudioOffload.IsEnabled)
```

Comment:
When `AudioOffload feature` is not supported, CheckDisabled() returns `InvalidOperationException` with 'offload is enabled' message, not `NotSupportedException`.
Is it proper exception and message?

Reply:
i put this code to check if target has the AudioOffload feature and it is enabled at the same time.
when CheckDisabled is called, it doesnt mean an user calls AudioOffload API directly.
CheckDisabled is called in other Audio APIs to use them.
let's say an user wants to call SetPlaybackRate. even if that target doesnt have the feature, SetPlaybackRate has to be called without any problems.
otherwise, if AudiOOffload exists and is enabled the user can't use this API.